symbolic-demangle/src/lib.rs

```diff
@@ -2,9 +2,9 @@
 //!
 //! Currently supported languages are:
 //!
-//! - C++ (GCC-style compilers and MSVC)
-//! - Rust (both `legacy` and `v0`)
-//! - Swift (up to Swift 5.2)
+//! - C++ (GCC-style compilers and MSVC) (`feature = "cpp" / "msvc"`)
+//! - Rust (both `legacy` and `v0`) (`feature = "rust"`)
+//! - Swift (up to Swift 5.2) (`feature = "swift"`)
```

Comment:
We can use Cargo.toml syntax here:

```suggestion
//! - C++ (GCC-style compilers and MSVC) (`features = ["cpp", "msvc"]`)
//! - Rust (both `legacy` and `v0`) (`features = ["rust"]`)
//! - Swift (up to Swift 5.2) (`features = ["swift"]`)
```